### ISSUE TITLE
Metronome example

### ISSUE THREAD

**Issue by eddysalzmann** [AudioKit/AudioKit#2336]
Hey there,

I'm pretty new to AudioKit and try to get my head around the AppleSequencer. In the examples, there is a "Drum Sequencer" that, if I could change the Audio files, would do the job. What I did is just drag and dropping the new Audio file into the Samples folder and reference that to eg the Hi Hat track. But when i play the file i hear only a dull bass drum sound. Also just swapping the existing files, eg a bass drum with a hi hat audio reference, doesn't work. I think I'm missing some basic knowledge about the functionalities. Here is what i did inside DrumSequencer.swift:

```
do {

            let metroURL = Bundle.main.resourceURL?.appendingPathComponent("Samples/metronome.wav")
            let metroFile = try AVAudioFile(forReading: metroURL!)

            try drums.loadAudioFiles([metroFile])

        } catch {
            Log("Files Didn't Load")
        }
        sequencer.clearRange(start: Duration(beats: 0), duration: Duration(beats: 100))
        sequencer.debug()
        sequencer.setGlobalMIDIOutput(drums.midiIn)
        sequencer.enableLooping(Duration(beats: 4))
        sequencer.setTempo(120)


        for i in 0 ... 7 {
            sequencer.tracks[0].add(
                noteNumber: 30,
                velocity: 127,
                position: Duration(beats: Double(i)),
                duration: Duration(beats: 1))
        }
```

Basically removed everything to just use the new sound. Why isn't that working?

Thanks a lot!

**Comment by aure** [comment]
Please repost to cookbook as there are no examples in AUdioKit anymore.

**Comment by boring2** [comment]
May I ask if this problem has been solved？
